src/components/dashboard/container/DictionariesDisplay.jsx

```diff
@@ -4,117 +4,132 @@ import propTypes from 'prop-types';
 import InfiniteScroll from 'react-infinite-scroll-component';
 import { connect } from 'react-redux';
 import '../styles/index.css';
-import { fetchDictionaries, searchDictionaries, fetchDictionary } from '../../../redux/actions/dictionaries/dictionaryActionCreators';
+import {
+  fetchDictionaries,
+  searchDictionaries,
+  fetchDictionary,
+} from '../../../redux/actions/dictionaries/dictionaryActionCreators';
 import { clearDictionaries } from '../../../redux/actions/dictionaries/dictionaryActions';
 import SideBar from '../components/SideNavigation';
 import ListDictionaries from '../components/dictionary/ListDictionaries';
 import SearchDictionaries from '../components/dictionary/DictionariesSearch';
 import UserDashboard from '../components/dictionary/user/UserDashboard';
+import AddDictionary from '../components/dictionary/AddDictionary';
 
 export class DictionaryDisplay extends Component {
-    static propTypes = {
-      fetchDictionaries: propTypes.func.isRequired,
-      dictionaries: propTypes.arrayOf(propTypes.shape({
-        dictionaryName: propTypes.string,
-      })).isRequired,
-      isFetching: propTypes.bool.isRequired,
-      searchDictionaries: propTypes.func.isRequired,
-      clearDictionaries: propTypes.func.isRequired,
-      fetchDictionary: propTypes.func.isRequired,
-      organizations: propTypes.arrayOf.isRequired,
+  static propTypes = {
+    fetchDictionaries: propTypes.func.isRequired,
+    dictionaries: propTypes.arrayOf(propTypes.shape({
+      dictionaryName: propTypes.string,
+    })).isRequired,
+    isFetching: propTypes.bool.isRequired,
+    searchDictionaries: propTypes.func.isRequired,
+    clearDictionaries: propTypes.func.isRequired,
+    fetchDictionary: propTypes.func.isRequired,
+    organizations: propTypes.arrayOf.isRequired,
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchInput: '',
+      dictionaryViewOptions: 'user',
+      show: false,
```

Comment:
Hi @hadijahkyampeire 
This state variable naming is somehow vague.
Just by looking at it, I am unable to guess what you are trying to show/hide, but after digging into the codebase I discovered that it is used to toggle the display of the Dictionary Modal.
A variable naming like `showDictionaryModal` would be better, the purpose of the variable is clear from its naming.

Reply:
Thanks for that but I think that will be done in another PR for refactoring code since this was for just moving the button and its properties to another page

Reply:
Owk. That's awesome.